ddl-tpch/bin_flat/alltables.sql

```diff
@@ -1,96 +0,0 @@
-create database if not exists ${DB};
-use ${DB};

```

Comment:
I will move it under bin_partitioned folder and rename it to allexternaltable.sql.